src/app/components/admin/admin-home/admin-home.component.ts

```diff
@@ -13,10 +13,10 @@ export class AdminHomeComponent implements OnInit {
   ) { }
 
   ngOnInit() {
-    this.getUser();
+    this.userRef();
   }
 
-  getUser() {
+  userRef() {
```

Comment:
I agree this one should be renamed but what we're doing here is subscribing to the adminUser$ observable, rather than "getting" or creating a reference. Let's call all similar methods watchSomething(). This one should be called watchUser() or even watchAdminUser(). Please do the same for similar methods which are actually subscribing to an observable, and especially those subscribing to a BehaviorSubject